### PR TITLE
Update Taxpasta to v0.6.0

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
Update TAXPASTA to the latest version, v0.6.0, which enables error handling for empty profiles when executing taxpasta merge.
## PR checklist
- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#323](https://github.com/nf-core/taxprofiler/pull/323) Add ability to skip sequencing quality control tools (❤️ to @vinisalazar for requesting, added by @jfy133)
 - [#345](https://github.com/nf-core/taxprofiler/pull/345) Add simple tutorial to explain how to get up and running with an nf-core/taxprofiler run (added by @jfy133)
 - [#355](https://github.com/nf-core/taxprofiler/pull/355) Add support for TAXPASTA's `--add-rank-lineage` to output (❤️ to @MajoroMask for request, added by @Midnighter, @sofstam, @jfy133)
+- [#368](https://github.com/nf-core/taxprofiler/pull/368/) Add the ability to ignore profile errors caused by empty profilers when merging multiple profiles using TAXPASTA(added by @@Midnighter and @LilyAnderssonLee)
```
Comment (jfy133) [inline CHANGELOG.md:22]:
```suggestion
- [#368](https://github.com/nf-core/taxprofiler/pull/368/) Add the ability to ignore profile errors caused by empty profilers when merging multiple profiles using TAXPASTA (added by @Midnighter and @LilyAnderssonLee)
```

**Review by jfy133 — APPROVED** [review]

File: nextflow_schema.json [hunk 2]
```diff
@@ -641,6 +641,12 @@
                     "description": "Add the taxon's entire rank lineage to the output.",
                     "help_text": "\nThe standard output format of taxpasta is a two-column table including the read counts and the integer taxonomic ID. The taxonomic ranks categories of the taxon's entire lineage separated by semi-colons can be added as additional information to the output table. This complements `--taxpasta_add_lineage` by telling you which taxonomic rank level each entry in the lineage refers to.\n\nModifies tool parameter(s):\n- taxpasta: `--add-rank-lineage`\n",
                     "fa_icon": "fas fa-link"
+                },
+                "taxpasta_ignore_errors": {
+                    "type": "boolean",
+                    "description": "Ignore individual profiles that cause errors.",
+                    "help_text": "\nIgnore any metagenomic profilers with errors by running `taxpasta merge`. At least two profilers without errors to merge.\n\nModifies tool parameter(s):\n- taxpasta: `--ignore-errors`\n",
```
Comment (Midnighter) [inline nextflow_schema.json:648]:
```suggestion
                    "help_text": "\nIgnore any metagenomic profiles with errors when running `taxpasta merge`. At least two profiles without errors are needed to merge.\n\nModifies tool parameter(s):\n- taxpasta: `--ignore-errors`\n",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#323](https://github.com/nf-core/taxprofiler/pull/323) Add ability to skip sequencing quality control tools (❤️ to @vinisalazar for requesting, added by @jfy133)
 - [#345](https://github.com/nf-core/taxprofiler/pull/345) Add simple tutorial to explain how to get up and running with an nf-core/taxprofiler run (added by @jfy133)
 - [#355](https://github.com/nf-core/taxprofiler/pull/355) Add support for TAXPASTA's `--add-rank-lineage` to output (❤️ to @MajoroMask for request, added by @Midnighter, @sofstam, @jfy133)
+- [#368](https://github.com/nf-core/taxprofiler/pull/368/) Add the ability to ignore profile errors caused by empty profilers when merging multiple profiles using TAXPASTA (added by @Midnighter and @LilyAnderssonLee)
```
Comment (Midnighter) [inline CHANGELOG.md:22]:
```suggestion
- [#368](https://github.com/nf-core/taxprofiler/pull/368/) Add the ability to ignore profile errors caused by empty profiles and other validation errors when merging multiple profiles using TAXPASTA (added by @Midnighter and @LilyAnderssonLee)
```

**Review by Midnighter — COMMENTED** [review]
Some recommended wording changes.

**Pull request merged**